Projects/1.Counter/index.css

```diff
@@ -0,0 +1,69 @@
+body{
+    background-color: lightblue;
+}
+.pointer{
+    cursor: pointer;
+}
+.div{
```

Comment:
we cannot use .div to define css. Lets assign some classname for that div and use it here.